frontend/js/behaviors/core/primaryNavigation.js

```diff
@@ -33,6 +51,18 @@ export default function(container) {
         document.removeEventListener(event.type, collapseHandler)
     }
 
+    function exhibitionHandler(json) {
+        let data = json['data'][0]
+
+        const template = container.querySelector('#menu-featured-exhibition').content.cloneNode(true)
+        template.querySelector('a').setAttribute('href', data.web_url)
+        template.querySelector('img').setAttribute('src', data.image_url)
+        template.querySelector('.title').textContent = data.title
+
+        exhibitionsDetails.querySelector('a').remove()
```

Comment:
What link is being removed here?

Reply:
This replaces [this link](https://github.com/art-institute-of-chicago/artic.edu/blob/feature/mega-menu/resources/views/partials/_navigation-list.blade.php#L44-L49) (which is set to [this url](https://github.com/art-institute-of-chicago/artic.edu/blob/feature/mega-menu/app/Providers/AppServiceProvider.php#L292)) with a link to the featured exhibition.